app/views/katello/api/v2/content_views/base.json.rabl

```diff
@@ -15,7 +15,7 @@ end
 
 node :last_published do |content_view|
   unless content_view.versions.empty?
-    content_view.versions.last.created_at
+    content_view.versions.order(:major).last.created_at
```

Comment:
This will ignore minor versions (which are generated by an incremental update).  You could do:

`content_view.versions.order(:major).order(:minor).last.created_at`
or just:
`content_view.versions.order(:created_at).last.created_at`